src/main/java/com/redfin/sitemapgenerator/GoogleCodeSitemapGenerator.java

```diff
@@ -80,6 +80,19 @@ public void render(GoogleCodeSitemapUrl url, OutputStreamWriter out,
 		public String getXmlNamespaces() {
 			return "xmlns:codesearch=\"http://www.google.com/codesearch/schemas/sitemap/1.0\"";
 		}
+

```

Comment:
It looks like all of the "weird" sitemap generators (code, video, etc.) now have two copies of the renderer: one OutputStreamWriter, and another for StringBuilder. We shouldn't leave two copies of the renderer around. Can we delete the OSW version and rely on AbstractSitemapUrlRenderer to call through to the SB version?